Tasks/GradleV2/Tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Agent.Tasks.Gradle",
+  "version": "2.139.1",
+  "author": "Microsoft Corporation",
+  "description": "Build with Gradle",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vso-agent-tasks/issues"
+  },
+  "dependencies": {
+    "vsts-task-lib": "2.6.0",
```

Comment:
Do you need this and `mocha`?  For my tests I've only needed `@types/mocha` in the Tests/package.json

Reply:
We have a dependency on the vsts-task-lib itself in the tests.

Reply:
That's usually the case but we haven't needed to install it twice.  Does it break if you uninstall it for Tests?

Reply:
Yes, it breaks when I remove the dependency.